ios/AirGoogleMaps/GlobalVars.m

```diff
@@ -267,11 +309,24 @@ - (UIImage *)getSharedUIImage:(NSString *)imageSrc {
         NSString *trafficValue = jsonDictionary[@"traffic"];
         NSString *zoomLevelValue = jsonDictionary[@"zoomLevel"];
         BOOL hasValetService = [jsonDictionary[@"hasValetService"] boolValue];
-
+        NSDictionary *planned = (NSDictionary *)jsonDictionary[@"planned"];
+        NSDictionary *maintenance = (NSDictionary *)jsonDictionary[@"maintenance"];
```

Comment:
what happens if maintenance doesnt exist? I think we should also make this part of code null safe to be backward compatible.